---a/net/minecraft/network/play/INetHandlerPlayClient.java
+++b/net/minecraft/network/play/INetHandlerPlayClient.java
@@ -73,8 +73,8 @@
 import net.minecraft.network.play.server.S48PacketResourcePackSend;
 import net.minecraft.network.play.server.S49PacketUpdateEntityNBT;
 
-public interface INetHandlerPlayClient extends INetHandler
-{
+public interface INetHandlerPlayClient extends INetHandler {
+
     void handleSpawnObject(S0EPacketSpawnObject packetIn);
 
     void handleSpawnExperienceOrb(S11PacketSpawnExperienceOrb packetIn);
@@ -216,4 +216,4 @@
     void handleResourcePack(S48PacketResourcePackSend packetIn);
 
     void handleEntityNBT(S49PacketUpdateEntityNBT packetIn);
-}
+}
